plugin.py

```diff
@@ -101,14 +105,16 @@ def onHeartbeat(self):
         now = datetime.now()
         if now >= self.nextupdate:
             self.nextupdate = now + timedelta(minutes=self.pollinterval)
-            data = json.loads(urllib.request.urlopen(
-                "http://api.wunderground.com/api/" + Parameters["Mode1"] + "/astronomy/q/" + Parameters["Mode2"] + "/" +
-                Parameters["Mode3"] + ".json").read().decode('ascii'))
+
+            u =  "http://api.wunderground.com/api/%s/astronomy/q/%s/%s.json"  % (Parameters["Mode1"],Parameters["Mode2"],Parameters["Mode3"])
+            data = json.loads(urllib.request.urlopen(u).read().decode('ascii'))
+            Domoticz.Log('Moon URL:%s' % u)
+
             lune = data['moon_phase']['phaseofMoon']
             luneage = data['moon_phase']['ageOfMoon']
-			# Domoticz.Log(str(lune))
```

Comment:
Those commented statements are useful when you test things. Not necessary but wont cause problems either!!